db/R/utils.R

```diff
@@ -107,17 +107,17 @@ db.open <- function(params) {
 ##' \dontrun{
 ##' db.close(con)
 ##' }
-db.close <- function(con) {
+db.close <- function(con, showWarnings=TRUE) {
```

Comment:
This is related to adding a `on.exit()` statement to met.process ([here](https://github.com/PecanProject/pecan/pull/1016/files#diff-a796467dc73dfb5ca85a17b225f8f409R23). I think that should be used in more places so that connections are always closed, even if there are errors. But if you do that but someone's explicitly closed the connection in the meantime you get a warning, so I added this option to suppress it.